CONTRIBUTING.md

```diff
@@ -53,7 +53,7 @@ included with the version update. The following is an example update to version
 - Add `config_managed` attribute to the `RoleBinding` type.
 ```
 
-Submit an MR with the CHANGES.md modification and review/merge.
+Submit an MR with the CHANGES.md and version.go modifications and review/merge.
```

Comment:
There is no version.go file in the ocm-api-model. Its only needed in the SDK. So we can remove these changes from the MR

Reply:
My brain thought this was under the ocm-sdk changes - good catch. Fixed.